README.md

```diff
@@ -1 +1,59 @@
-# HELLO-MICROVERSE
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# hello microvese
+
+> hello microverse project.
+
+
+## Built With
+
+- html/css
+
+## Getting Started
+
+**This is a microverse hello project.**
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```

Comment:
- [ ] As you can see there is no prerequisite in order to run this project. Please Consider removing :x:  this line to keep only the necessary steps in your README. Removing this line will make your README looks more professional :1st_place_medal: 
The same applies to  Run tests and Deployment headlines. 
For the remaining headlines, Please add the necessary steps. In case there is nothing to add, my suggestion is to remove any headline which has no content
Your section should be similar to something like this :arrow_down_small:

To get a local copy up and running follow these simple example steps.
### Setup
Clone this repository or download the Zip folder:
### Install
`git clone <add the link to your repo  here>`
## Usage
Navigate to the location of the folder in your machine:

`you@your-Pc-name:~$ cd <folder>
`
    Press Enter to navigate to your local clone.